forwards_paper.tex

```diff
@@ -147,38 +151,37 @@ \section*{Introduction}
 for many biological population sizes of interest.
 
 However, it is thought that most genetic variation is selectively neutral (or nearly so).
-By definition, the neutral alleles carried by individuals in a population
+By definition, neutral alleles carried by individuals in a population
 do not affect the population process.
 For this reason, if one records the entire genealogical history of a population over the course of a simulation,
 simply laying down neutral mutations on top of that history afterwards
-is equivalent to having generated them during the simulation.
-Precisely, we would need to know the genealogical tree relating all sampled individuals
+is equivalent to having generated them during the simulation:
+it does not matter if we generate each generation's mutations during the simulation, or afterwards.
+Precisely, we need to know the genealogical trees relating all sampled individuals
 at each position along the genome.
-This is equivalent to knowing the \emph{population pedigree} --
-the complete history of parent-offspring relationships of an entire population
-going back to a remote time -- and the genetic outcomes of each ancestral
-meiosis.
-The embellished pedigree contains all the information necessary
-to construct the genealogical tree that relates each individual to every other
-at each position on the genome,
-i.e., the \emph{tree sequence} implemented in the coalescent simulator \msprime{}~\citep{kelleher2016efficient}.
```

Comment:
I rearranged this bit to (hopefully) flow more logically, separating "the embellished population pedigree" from "the tree sequence".